pkg/bot/interactive/help.go

```diff
@@ -81,6 +82,14 @@ func Help(platform config.CommPlatformIntegration, clusterName, botName string)
 					},
 				},
 			},
+			{
+				Base: Base{
+					Description: "Angry? Amazed?",
+				},
+				Buttons: []Button{
+					btnBuilder.DescriptionURL("Give feedback", "feedback", "https://feedback.botkube_io", slack.StylePrimary),
```

Comment:
```suggestion
					btnBuilder.DescriptionURL("Give feedback", "feedback", "https://feedback.botkube.io", slack.StylePrimary),
```

please also fix it for twitter in line: 97. Thanks!